### ISSUE TITLE
"No such attribute" error on reference to inner enum member named "get"

### ISSUE THREAD

**Issue by kingjon3377** [ceylon/ceylon-ide-intellij#669]
The IntelliJ IDE fails to compile a line of code referring to the `get` (`GET` in Java) member of the `Method` enum in the `NanoHTTPD` class in [NanoHTTPD/nanohttpd](https://github.com/NanoHTTPD/nanohttpd) (which I imported as `import maven:"org.nanohttpd:nanohttpd" "2.3.1"`), claiming "method or attribute is not defined: get in type Method might be misspelled". The code compiles nearly cleanly (a "duplicate class" backend error in an entirely different module when I compile my whole project in one pass, but no errors when compiling either module individually) under the command-line compiler.

**Comment by gavinking** [comment]
How did you write the code in Ceylon?

    import fi.iki.elonen { NanoHTTPD { Method { get } } }

Or:

    import fi.iki.elonen { NanoHTTPD { Method } }

    Method.get

Or what?

Does it work for `Method.put`?

**Comment by gavinking** [comment]
Also, I think this bug is mis-filed. Did you really mean to report it against `ceylon-ide-intellij`?

**Comment by kingjon3377** [comment]
I tried both `import fi.iki.elonen { NanoHTTPD }`, referring to the value as either `NanoHTTPD.Method.get` or (within a subclass of `NanoHTTPD`) `Method.get`, and `import fi.iki.elonen { NanoHTTPD { Method } }`; I don't think I tried `import fi.iki.elonen { NanoHTTPD { Method { get } } }`. (I didn't try `Method.put`; I was only referring to `Method.get` to reject all requests using any other verb.)

But in any case I now can't reproduce this problem. In my code, the "duplicate class" backend error when using the CLI compiler still happens with kingjon3377/strategicprimer-viewer@3d0cf2a, but *not* when I rebase it on top of my current version (the only difference being a switch from [brettwoodridge/SparseBitSet](https://github.com/brettwooldridge/SparseBitSet) to [RoaringBitmap/RoaringBitmap](https://github.com/RoaringBitmap/RoaringBitmap) in a module this one indirectly imports); I'll try to reduce my code to a more minimal example that will still produce that error and report it against `eclipse/ceylon`. But even checking out kingjon3377/strategicprimer-viewer@3d0cf2a, replacing the workaround for this problem I included in that commit with the code that triggered this, and forcing recompilation still doesn't trigger this error. So I'll close this as unreproducible.
